x-pack/platform/plugins/shared/alerting/server/saved_objects/schemas/raw_rule/v5.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { rawRuleSchema as rawRuleSchemaV4 } from './v4';
+
+export const dashboardsSchema = schema.arrayOf(schema.object({ id: schema.string() }));
```

Comment:
Same dashboardsSchema is defined [here](https://github.com/elastic/kibana/pull/209668/files#diff-19b19826913ccad617e1a12cd27531edd7fe2068ba51532ebfc8fd5dda064ae4R152) as well. I noticed that rest saved_objects schemas, for example [flappingSchema](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/alerting/server/saved_objects/schemas/raw_rule/v2.ts#L12), is re-defined in the saved_objects folder and not being reused from the flappingSchema defined [here](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/alerting/server/application/rule/schemas/flapping_schema.ts#L10). That's why I followed same pattern here and duplicated the schema one more time here. Is there any specific reason we don't reuse schemas?